src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/SharedLiveMedia/SharedLiveMediaItem/index.tsx

```diff
@@ -68,48 +75,50 @@ export const SharedLiveMediaItem = ({
         />
       </Box>
 
-      <Box
-        align="center"
-        direction="row"
-        justify="center"
-        margin={{ left: 'auto' }}
-      >
-        {sharedLiveMedia.upload_state === uploadState.READY ? (
-          <Box justify="center" margin={{ left: 'auto' }}>
-            {!isShared ? (
-              <StartSharingButton sharedLiveMediaId={sharedLiveMedia.id} />
-            ) : (
-              <StopSharingButton />
-            )}
-          </Box>
-        ) : IS_UPLOAD_IN_PROGRESS ? (
-          <Text
-            color="blue-active"
-            size="0.9rem"
-            style={{ fontFamily: 'Roboto-Medium' }}
-            truncate
-          >
-            <ObjectStatusPicker object={sharedLiveMedia} />
-          </Text>
-        ) : (
-          <React.Fragment>
-            <Box>
-              <Text
-                color="red-active"
-                size="0.9rem"
-                style={{ fontFamily: 'Roboto-Medium' }}
-              >
-                {intl.formatMessage(messages.retryUploadFailedLabel)}
-              </Text>
+      {isLive && isTeacher && (
```

Comment:
in fact only the `share` button should be hidden when it's not a live. All other case should be shown.